spdm.rflx

```diff
@@ -0,0 +1,1487 @@
+   --  that is selected by a refinement.
+   type Challenge_Auth_Response_1_0 is
+      message
+         Slot            : Slot;
+         Slot_Mask       : Slot_Mask;
+         Cert_Chain_Hash : Hash;
+         Nonce           : Opaque
+            with Size => 8 * 32;
+      end message;
+
+   type Challenge_Auth_Response_1_1 is
+      message
+         Basic_Mut_Auth_Req : Boolean;
+         Reserved           : Reserved_3;
+         Slot               : Narrow_Slot;
+         Slot_Mask          : Slot_Mask;
+         Cert_Chain_Hash    : Hash;
+         Nonce : Opaque
+            with Size => 8 * 32;
+         --  ISSUE: Componolit/systematization-binary-schemas#53
+         --  This field shall
+         --    - not be present if Hash_Type of the request was No_Measurement or measurement is
+         --      unsupported
+         --    - contain a hash if Hash_Type was TCB_Measurement
+         --    - be 0 if Hash_Type was TCB_Measurement and no measurement is required
+         --    - contain a hash of all measurements if Hash_Type was All_Measurments
+         Measurement_Summary_Hash : Hash;
+         Opaque_Length_1 : Length;
+         Opaque_Length_2 : Length
+            then Opaque_Data
+               if Opaque_Length_2 * 256 + Opaque_Length_1 <= 1024;
+         Opaque_Data : Opaque
+            with Size => 8 * (Opaque_Length_2 * 256 + Opaque_Length_1);
+         Signature : Signature;
+      end message;
+
+   -- 10.12 ERROR response message
+
+   type Error_Code is
+      (Invalid_Request        => 16#01#,
+       Invalid_Session        => 16#02#,
+       Busy                   => 16#03#,
+       Unexpected_Request     => 16#04#,
+       Unspecified            => 16#05#,
+       Decrypt_Error          => 16#06#,
+       Unsupported_Request    => 16#07#,
+       Request_In_Flight      => 16#08#,
+       Invalid_Response_Code  => 16#09#,
+       Session_Limit_Exceeded => 16#0A#,
+       Major_Version_Mismatch => 16#41#,
+       Response_Not_Ready     => 16#42#,
+       Request_Resynch        => 16#43#,
+       Vendor                 => 16#FF#)
+   with Size => 8;
+
+   type Error_Session_ID is mod 2**8;
+
+   type RDT_Exponent is mod 2**8;
+   type Token is mod 2**8;
+   type RDTM is range 1 .. 2**8 - 1 with Size => 8;
+
+   type Response_Not_Ready_Data is
+      message
+         RDT_Exponent : RDT_Exponent;
+         Request_Code : Request_Code;
+         Token        : Token;
+         RDTM         : RDTM;
+      end message;
+
+   type Zero_Error_Data is range 0 .. 0 with Size => 8;
+
+   type Vendor_ID_Len is mod 2**8;
+   type Vendor_ID_Double is mod 2**16;
+   type Vendor_ID_Quad is mod 2**32;
+
+   --  ISSUE: Componolit/RecordFlux#609
+   --  This message differs between SPDM version 1.0 and 1.1.
+   --  Since the minor version is not accessible from
+   --  within this message one message is needed for each version
+   --  that is selected by a refinement.
```

Comment:
As above.

Reply:
Componolit/RecordFlux#609 is fixed. This actually is part of #92 where the duplicate message should be merged again.

Reply:
Fixed in https://github.com/Componolit/RecordFlux-specifications/pull/93/commits/7c1908a0db7eece9cbbb987fa14b0c9d9e4ec91f.